### ISSUE TITLE
Files with ACLs or extended security information incorrectly considered modified

### ISSUE THREAD

**Issue by ajoberstar** [ajoberstar/grgit#69]
See ajoberstar/gradle-git#124 for details. This will have to be resolved in JGit.

**Comment by jzwolak** [comment]
Perhaps we should update grgit to use the latest jgit and see if the behavior is still present.  I'm going to look into doing this right now and see how it goes.

**Comment by jzwolak** [comment]
I built and tested grgit with jgit 4 successfully.  All I did was change this line:

    def jgitVersion = '4.0.1.201506240215-r'

And the ACL problem is not present in this version of jgit.

Also, if you're trying to reproduce the problem it is subtler than I thought.  The problem occurs because the ACL has the executable permission and the unix file permissions do not.  Git on the command line was using the unix file permissions while jgit was using the ACL permissions.  jgit 4 must be using the unix file permissions since it does not show the file as different when I set up this problem and simply change the version of jgit from 3 to 4 through a forced transitive dependency in gradle.

Please update the version of jgit used by grgit and this problem will be fixed.

**Comment by radimk** [comment]
Indeed, newer jgit fixes this problem for me. I only needed to update my buildscript prelude to 
```
buildscript {
    ...
    dependencies {
        classpath 'org.ajoberstar:gradle-git:1.2.0'
        classpath 'org.eclipse.jgit:org.eclipse.jgit:4.0.1.201506240215-r'
        ...
    }
}
And a symlinks that was previously seen as unstaged change is OK now.

**Comment by jzwolak** [comment]
That's correct radimk.  I also tested your solution.  This will work if Gradle's default dependency resolution is being used (use the newest version when multiple versions are specified).  For users who've changed the dependency resolution this won't work.

**Comment by ajoberstar** [comment]
Typed the wrong ID into the commit message, but fixed as of: bd7083a0d46e67a3eede5c076ba98488b215ab67
